test/lit/passes/outlining.wast

```diff
@@ -2,41 +2,54 @@
 
 ;; RUN: foreach %s %t wasm-opt --outlining -S -o - | filecheck %s
 
+;; TODO: Add a test that creates an outlined function with a sequence at beginning
+;; TODO: Add a test that creates an outlined function with one return value
+;; TODO: Add a test that creates an outlined function that no arguments
+;; TODO: Add a test that creates an outlined function that returns multiple values
+;; TODO: Add a test that makes sure we filter localSets correctly
+;; TODO: Add a test that makes sure we filter localGets correctly
+;; TODO: Add a test that makes sure we filter branches correctly
+;; TODO: Add a test that makes sure we filter globals correctly
+;; TODO: Add a test that fails to outline a single control flow that repeats
+
+
 (module
-	;; CHECK:      (type $0 (func (result i32)))
+  ;; CHECK:      (type $0 (func (result i32)))
 
-	;; CHECK:      (type $1 (func (param i32)))
+  ;; CHECK:      (type $1 (func (param i32)))
 
-	;; CHECK:      (func $a (result i32)
-	;; CHECK-NEXT:  (call $outline$5
-	;; CHECK-NEXT:   (i32.const 7)
-	;; CHECK-NEXT:  )
-	;; CHECK-NEXT:  (return
-	;; CHECK-NEXT:   (i32.const 4)
-	;; CHECK-NEXT:  )
-	;; CHECK-NEXT: )
-	(func $a (result i32)
-	  (drop (i32.const 7))
-	  (drop (i32.const 1))
-	  (drop (i32.const 2))
-	  (return (i32.const 4))
-	)
-	;; CHECK:      (func $b (result i32)
-	;; CHECK-NEXT:  (call $outline$5
-	;; CHECK-NEXT:   (i32.const 0)
-	;; CHECK-NEXT:  )
-	;; CHECK-NEXT:  (return
-	;; CHECK-NEXT:   (i32.const 5)
-	;; CHECK-NEXT:  )
-	;; CHECK-NEXT: )
-	(func $b (result i32)
-	  (drop (i32.const 0))
-	  (drop (i32.const 1))
-	  (drop (i32.const 2))
-	  (return (i32.const 5))
-	)
+  ;; CHECK:      (func $a (result i32)
+  ;; CHECK-NEXT:  (call $outline$5
+  ;; CHECK-NEXT:   (i32.const 7)
+  ;; CHECK-NEXT:  )
+  ;; CHECK-NEXT:  (return
+  ;; CHECK-NEXT:   (i32.const 4)
+  ;; CHECK-NEXT:  )
+  ;; CHECK-NEXT: )
+  (func $a (result i32)
+    (drop (i32.const 7))
+	(drop (i32.const 1))
```

Comment:
Indentation is still weird here.

Reply:
I believe the moral is never tab ever. vim totally didn't show this. Just deleted all the space and then manually hit the spacebar to fix 🤣